rfcs/20180920-unify-rnn-interface.md

```diff
@@ -0,0 +1,356 @@
+# Unify RNN Inteface
+
+| Status        | Proposed                                             |
```

Comment:
Scott, 
if all the comments are addressed can you please update the status to "Accepted"

Reply:
Done.